types/react-relay/test/relay-experimental-tests.tsx

```diff
@@ -927,3 +928,51 @@ function QueryLoader() {
         }
     }
 }
+
+/**
+ * Tests for loadQuery
+ * see https://github.com/facebook/relay/blob/master/packages/relay-experimental/loadQuery.js
+ */
+function LoadQuery() {
+    interface AppQueryVariables {
+        id: string;
+    }
+
+    interface AppQueryResponse {
+        readonly user: {
+            readonly name: string;
+        };
+    }
+
+    interface AppQuery {
+        readonly response: AppQueryResponse;
+        readonly variables: AppQueryVariables;
+    }
+
+    const query = graphql`
+        query AppQuery($id: ID!) {
+            user(id: $id) {
+                name
+            }
+        }
+    `;
+
+    const variables: AppQueryVariables  = {
+        id: "1"
+    };
+
+    const dataWithoutExtraOptions = loadQuery<AppQuery>(environment, query, variables);
+
+    const dataWithOptions = loadQuery<AppQuery>(environment, query, variables, {
+        fetchPolicy: 'store-or-network',
+        networkCacheConfig: {
+            force: true
+        }
+    });
+
+    function ShouldPassIntoUsePreloadQuery() {
```

Comment:
Btw it seems likely that in reality the `preloadedQuery` argument would be a function parameter and not exist in the same module, like your example here. So I think the test should probably be rewritten like:

```suggestion
    function ShouldPassIntoUsePreloadQuery(preloadedQuery: PreloadedQuery<AppQuery>) {
```

Reply:
…and then update the below line to use the `preloadedQuery` argument instead of the `dataWithOptions` lvar.